cpp/arcticdb/entity/types.hpp

```diff
@@ -124,7 +124,7 @@ enum class ValueType : uint8_t {
     // For instance, Pandas empty series whose types has not been specified is mapping to EMPTY.
     // When data is appended, the column type is inferred from the data and the column is promoted to the inferred type.
     EMPTY = 13,
-    COUNT // Not a real value type, should not be added to proto descriptor. Used to count the number of items in the enum
+    COUNT = 14 // Not a real value type, should not be added to proto descriptor. Used to count the number of items in the enum
```

Comment:
NIT: I think this is a bit error prone. When adding new type people have to add value and update the count. Usually the count is left without a value so that it auto-increments